code/datums/sexcon/sexcon.dm

```diff
@@ -94,10 +94,13 @@
 	if(!user.defiant && !victim.defiant)
 		return FALSE
 	// Need to violate AFK clients
-	if(victim.mind && victim.mind.key && !victim.client)
+	if(!victim.mind || !victim.mind.key || !victim.client) // Changed to OR statements to remove ZAPE of mobs without minds or keys
 		return TRUE
-	// Need to violate combat mode people
-	if(victim.cmode)
+	// Need to violate combat mode people - Include user to remove ZAPE
+	if(victim.cmode || user.cmode)
+		return TRUE
+	// Need to violate unconscious people to remove ZAPE
+	if(victim.stat)
```

Comment:
Same as above- I think this is fine, I think it should only apply to defiant people.

Reply:
Correction: This only applies if either user or victim are defiant, due to the early return above